src/GrassPokemon.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class GrassPokemon extends Pokemon{
+    private final String type = "grass";
```

Comment:
Je kunt deze methode ook nog "static" maken, omdat het voor alle instances van GrassPokemon hetzelfde is (bulbasaur = grass, ivysaur = grass, oddish = grass, etc). 
Dit is in feite geen "variabele" maar een "constante". Conventie is in dat geval om het met hoofdletters te schrijven. Dus het zou dan worden:  `private final static String TYPE= "grass";`